src/euphorie/client/browser/templates/locking_menu.pt

```diff
@@ -31,7 +31,7 @@
                   datetime="${value}"
                   data-pat-display-time="from-now: true; locale: ${language}"
                   tal:define="
-                    value python:webhelpers.timezoned_date(last_locking_event.time);
+                    value python:webhelpers.timezoned_date(view.lock_time);
```

Comment:
FWIW on my machine the time is two hours off. It may of course depend on the locale settings. I'll investigate some more.

Reply:
The problem is that we use the default for the time column, which is `sqlalchemy.func.now()` and returns the current UTC time. We store that but later interpret it as a time in the server's time zone. This goes wrong if the server is in a differing time zone.

I've changed the code now so that locking and validation times are passed explicitly as `datetime`s like we do for the `modified` date. Then using `timezoned_date` makes sense, because the time is stored as server local time, only the zone information is lost, and then later restored when displaying the time.